foreman/data_refinery_foreman/surveyor/management/commands/survey_all.py

```diff
@@ -43,7 +43,7 @@ def handle(self, *args, **options):
                 s3 = boto3.resource('s3')
                 try:
                     filepath = "/tmp/input_" + str(uuid.uuid4()) + ".txt"
-                    s3.Bucket(bucket).download_file(key, "/tmp/input_" + str(uuid.uuid4()) + ".txt")
```

Comment:
I like this commit using a defined variable instead of having this written in two places, but I'm confused about how it relates to the topic of this PR.

Reply:
The code pre-change is actually broken since it's actually generating two separate uuids. It's not related to the other AE changes other than it allows to use S3-hosted accession lists when kicking off jobs. I can file a separate PR, but that seemed like overkill for a trivial 1-line fix.

Reply:
Ahh! Two separate uuids sound bad! I think it would be better to make this a separate PR because then it's not held up by anything else here & seems like a very fast merge. The two separate uuids bit sounds particularly urgent.